### PR TITLE
standby-for options have been deprecated within cephfs (BSC#1171365)

### PR DESCRIPTION
**Pull request by asettle**
Removing documentation for standby-for options and adding
upstream documentation.

### CODE REVIEW

File: xml/admin_ceph_cephfs.xml [hunk 1]
```diff
@@ -382,103 +382,31 @@ setfattr -n ceph.dir.pin -v 0 a/b # "a/b" is now pinned to rank 0
    more 'standby' daemons that will take over during the MDS daemon failover.
   </para>
 
-  <sect2 xml:id="ceph-cephfs-failover-standby">
-   <title>Configuring Standby Daemons</title>
+  <sect2 xml:id="ceph-cephfs-failover-standbyreplace">
```
Comment (jan--f) [inline xml/admin_ceph_cephfs.xml:385]:
```suggestion
  <sect2 xml:id="ceph-cephfs-failover-standby">
```
Typo?

File: xml/admin_ceph_cephfs.xml [hunk 2]
```diff
@@ -382,103 +382,31 @@ setfattr -n ceph.dir.pin -v 0 a/b # "a/b" is now pinned to rank 0
...
-    daemon loads these settings when it starts, and sends them to the monitor.
+    Each &cephfs; file system may be configured to add standby-replay daemons.
+    These standby daemons follow the active MDS's metadata journal to reduce
+    failover time in the event active MDS becomes unavailable. Each active
+    MDS may have only one standby-replay daemon following it.
    </para>
    <para>
-    By default, if none of these settings are used, all MDS daemons which do
-    not hold a rank will be used as 'standbys' for any rank.
+     Configure standby-replace on a file system with the following comamnd:
```
Comment (jan--f) [inline xml/admin_ceph_cephfs.xml:394]:
```suggestion
     Configure standby-replay on a file system with the following comamnd:
```

File: xml/admin_ceph_cephfs.xml [hunk 3]
```diff
@@ -383,102 +383,30 @@ setfattr -n ceph.dir.pin -v 0 a/b # "a/b" is now pinned to rank 0
...
-   <title>Configuring Standby Daemons</title>
+   <title>Configuring Standby Replay</title>
    <para>
-    There are several configuration settings that control how a daemon will
-    behave while in standby. You can specify them in the
-    <filename>ceph.conf</filename> on the host where the MDS daemon runs. The
-    daemon loads these settings when it starts, and sends them to the monitor.
+    Each &cephfs; file system may be configured to add standby-replay daemons.
+    These standby daemons follow the active MDS's metadata journal to reduce
+    failover time in the event active MDS becomes unavailable. Each active
```
Comment (lproven) [inline xml/admin_ceph_cephfs.xml:390]: "in the event that an active MDS",  or "in the event that the active MDS"?

Sounds better to me. Not sure of the exact meaning.

File: xml/admin_ceph_cephfs.xml [hunk 4]
```diff
@@ -383,102 +383,30 @@ setfattr -n ceph.dir.pin -v 0 a/b # "a/b" is now pinned to rank 0
...
+</screen>
    <para>
-    The settings which associate a standby daemon with a particular name or
-    rank do not guarantee that the daemon will only be used for that rank. They
-    mean that when several standbys are available, the associated standby
-    daemon will be used. If a rank is failed, and a standby is available, it
-    will be used even if it is associated with a different rank or named
-    daemon.
+     Once set the monitors will assign available standby daemons to follow
+     the active MDS's in that file system.
```
Comment (lproven) [inline xml/admin_ceph_cephfs.xml:401]:
```suggestion
     the active MDSs in that file system.
```

**Pull request merged**